resources/files/sysv-wrapper.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+unset LD_LIBRARY_PATH
+unset LD_PRELOAD
```

Comment:
It's prohibitive. I couldn't get it working in the LIBRARY_PATH test, and I ripped out the preload test entirely because it's way too complex to do effectively.